x-pack/legacy/plugins/ml/public/timeseriesexplorer/_timeseriesexplorer.scss

```
@@ -333,3 +333,9 @@
     }
   }
 }
+
+/* Hides the progress bar's background so it doesn't look like it increases the thickness
+   of the horizontal bar below the tab menu elements in its inactive state. */
+.mlTimeSeriesExplorerProgress { background-color: $euiColorEmptyShade; }
+.mlTimeSeriesExplorerProgress::-moz-progress-bar { background-color: $euiColorEmptyShade; }
+.mlTimeSeriesExplorerProgress::-webkit-progress-bar { background-color: $euiColorEmptyShade; }
```

for scss you may also use the following:
```suggestion
.mlTimeSeriesExplorerProgress {
  background-color: $euiColorEmptyShade; 
  
  &::-moz-progress-bar, &::-webkit-progress-bar {
    background-color: $euiColorEmptyShade;
  }
}
```